### PR TITLE
docs: OGM custom resolver util functions

### PR DESCRIPTION
**Pull request by tbwiss**
Based on internal Trello card.

## Complexity

> **Note**
>
> Please provide an estimated complexity of this PR of either Low, Medium or High

Complexity: Low

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-3209.surge.sh

File: docs/modules/ROOT/pages/ogm/examples/custom-resolvers.adoc [hunk 1]
```diff
@@ -124,6 +124,43 @@ Promise.all([neoSchema.getSchema(), ogm.init()]).then(([schema]) => {
 
 It's important to note the JWT secret being passed into the `Neo4jGraphQL` constructor in this example.
 
+Additionally, an example implementation for the util functions `createJWT` and `comparePassword` is provided in this code snippet:
+[source, javascript, indent=0]
+----
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
```
Comment (mjfwebb) [inline docs/modules/ROOT/pages/ogm/examples/custom-resolvers.adoc:131]: Perhaps we can use import here instead, now?

Comment (tbwiss) [reply docs/modules/ROOT/pages/ogm/examples/custom-resolvers.adoc:131]: I first had it as import statements but most other code snippets in this subfolder use `require` so I went with the majority. Bit of a mixed bag

Comment (mjfwebb) [reply docs/modules/ROOT/pages/ogm/examples/custom-resolvers.adoc:131]: OK, might be a thing to do as a sweeping change later then.

Comment (tbwiss) [reply docs/modules/ROOT/pages/ogm/examples/custom-resolvers.adoc:131]: I would argue so. First and foremost to keep consistency, especially within a single file/page.

**Review by mjfwebb — APPROVED** [review]

**Review by lidiazuin — APPROVED** [review]

**Pull request merged**